app/components/pages/createPoll/datePicker.js

```diff
@@ -0,0 +1,45 @@
+import 'date-fns';
+import React from 'react';
+import Box from '@material-ui/core/Box';
+import DateFnsUtils from '@date-io/date-fns';
+import { useField, useFormikContext } from 'formik';
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker,
+} from '@material-ui/pickers';
+
+import { useStyles } from './styles';
+
+const MaterialUIPickers = ({ label, ...props }) => {
+  const classes = useStyles();
+  const { setFieldValue } = useFormikContext();
+  const [field, meta] = useField(props);
+
+  return (
+    <MuiPickersUtilsProvider utils={DateFnsUtils}>
+      <Box>
+        <KeyboardDatePicker
+          {...field}
+          {...props}
+          disableToolbar
+          format="MM/dd/yyyy"
+          id="date-picker-inline"
+          label={label}
+          inputVariant="outlined"
+          KeyboardButtonProps={{
+            'aria-label': 'change date',
+          }}
+          value={field.value}
+          selected={field.value}
+          onChange={(val) => {
+            setFieldValue(field.name, val);
```

Comment:
do we really need to set values like this? i don't think so check muiInput we are not setting fields like this it do this automatically we are just required to pass name to this component

Reply:
before i was not aware of boilerplate, as on codesand box i was doing this way to achive functionality.